08-interop/README.md

```diff
@@ -281,3 +281,188 @@ guide on Zephyr's documentation.
+2020-08-02 20:45:02,921 #           inet6 addr: 2001:16b8:453f:1ff0:3e71:bfff:fe9e:13fd  scope: global  VAL
+2020-08-02 20:45:02,924 #           inet6 group: ff02::2
+2020-08-02 20:45:02,927 #           inet6 group: ff02::1
+2020-08-02 20:45:02,930 #           inet6 group: ff02::1:ff9e:13fd
+2020-08-02 20:45:02,931 #           
+2020-08-02 20:45:02,934 #           Statistics for Layer 2
+2020-08-02 20:45:02,937 #             RX packets 0  bytes 0
+2020-08-02 20:45:02,942 #             TX packets 0 (Multicast: 0)  bytes 214
+2020-08-02 20:45:02,945 #             TX succeeded 3 errors 0
+2020-08-02 20:45:02,947 #           Statistics for IPv6
+2020-08-02 20:45:02,950 #             RX packets 3  bytes 312
+2020-08-02 20:45:02,955 #             TX packets 4 (Multicast: 2)  bytes 264
+2020-08-02 20:45:02,958 #             TX succeeded 4 errors 0
+```
+
+You should be able to reach the border router using it's global address
+
+```
+2020-08-02 20:45:41,423 #  ping6 2001:16b8:453f:1ff0:ecfa:bcff:fe5f:8291
+2020-08-02 20:45:41,439 # 12 bytes from 2001:16b8:453f:1ff0:ecfa:bcff:fe5f:8291: icmp_seq=0 ttl=64 time=8.020 ms
+2020-08-02 20:45:42,438 # 12 bytes from 2001:16b8:453f:1ff0:ecfa:bcff:fe5f:8291: icmp_seq=1 ttl=64 time=7.252 ms
+2020-08-02 20:45:43,438 # 12 bytes from 2001:16b8:453f:1ff0:ecfa:bcff:fe5f:8291: icmp_seq=2 ttl=64 time=6.779 ms
+2020-08-02 20:45:43,438 # 
+2020-08-02 20:45:43,443 # --- 2001:16b8:453f:1ff0:ecfa:bcff:fe5f:8291 PING statistics ---
+2020-08-02 20:45:43,449 # 3 packets transmitted, 3 packets received, 0% packet loss
+2020-08-02 20:45:43,453 # round-trip min/avg/max = 6.779/7.350/8.020 ms
+```
+
+You should be able to reach a global address on the internet
+
+```
+2020-08-02 20:45:49,052 #  ping6 2600::
+2020-08-02 20:45:49,200 # 12 bytes from 2600::: icmp_seq=0 ttl=49 time=143.513 ms
+2020-08-02 20:45:50,200 # 12 bytes from 2600::: icmp_seq=1 ttl=49 time=143.223 ms
+2020-08-02 20:45:51,199 # 12 bytes from 2600::: icmp_seq=2 ttl=49 time=142.450 ms
+2020-08-02 20:45:51,200 # 
+2020-08-02 20:45:51,202 # --- 2600:: PING statistics ---
+2020-08-02 20:45:51,207 # 3 packets transmitted, 3 packets received, 0% packet loss
+2020-08-02 20:45:51,212 # round-trip min/avg/max = 142.450/143.062/143.513 ms
+``` 
+
+DNS names should also get resolved
+
+```
+2020-08-02 20:45:57,277 #  ping6 riot-os.org
+2020-08-02 20:45:57,330 # 12 bytes from 2a01:4f8:151:64::11: icmp_seq=0 ttl=55 time=35.999 ms
+2020-08-02 20:45:58,335 # 12 bytes from 2a01:4f8:151:64::11: icmp_seq=1 ttl=55 time=40.993 ms
+2020-08-02 20:45:59,328 # 12 bytes from 2a01:4f8:151:64::11: icmp_seq=2 ttl=55 time=34.089 ms
+2020-08-02 20:45:59,329 # 
+2020-08-02 20:45:59,331 # --- riot-os.org PING statistics ---
+2020-08-02 20:45:59,336 # 3 packets transmitted, 3 packets received, 0% packet loss
+2020-08-02 20:45:59,340 # round-trip min/avg/max = 34.089/37.027/40.993 ms
+```
+
+And finally, you should be able to reach the 6LoWPAN node from any IPv6 host behind your firewall
```

Comment:
Where does the firewall come from? It was not mentioned before.

Reply:
The router will usually have one so IPv6 hosts in the LAN can't be reached from the Internet.
It *should* not block ICMPv6, but it often does.

So someone might be clever and try to ping the node from a host on the Internet and find it not working, might have to word that better.

Reply:
Either say "from your local network" or explain what you mean by "behind your firewall". Everything else might lead to confusion.

Reply:
Ok!